lib/scrabble/player.rb

```diff
@@ -0,0 +1,122 @@
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :tiles
+
+
+    ## -------------------------------------------------------------------------
+    # CLASS METHODS
+
+
+    def initialize(name)
+      @bag_o_tiles = TileBag.new
+      @name = name
+      @plays = []
+      @tiles = []
+
+      draw_tiles(@bag_o_tiles)
+    end
+
+
+    ## -------------------------------------------------------------------------
+    # INSTANCE METHODS
+
+
+    def play(word)
+      unless won?
+      # do this stuff inside _unless player has already won_.
+      # then return true to indicate that the word was played. :)
+
+        score = Scrabble.score(word)
+
+        if (score.class == Fixnum)
```

Comment:
Clever. You will learn other ways to deal with this case (invalid input) later on.